elastica/modules/base_system.py

```diff
@@ -181,22 +183,22 @@ def finalize(self):
                 self._feature_group_synchronize.pop(index)
             )
 
-    def synchronize(self, time: float):
+    def synchronize(self, time: np.floating):
```

Comment:
why np.floating? instead of float. Also there are np.float64 or np.float32, is np.floating more general

Reply:
np.floating includes all size of float. Python float and np.float ling is different, and I don't have exact justification other than numpy has numerically accurate representation where python-float has better print